lib/veri_report.ml

```diff
@@ -0,0 +1,56 @@
+open Core_kernel.Std
+open Bap.Std
+open Bap_traces.Std 
+
+module Map = String.Map
+
+type hist = (string * int) list [@@deriving bin_io, sexp]
+
+type succ = [ 
```

Comment:
I want to refine this module. with usage of `Veri_frame` module. And create `Veri_report` with
test function `frame -> bool`